sigal/plugins/watermark.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+"""Plugin which adds a watermark to the image.
+
+Settings:
+
+- ``watermark``: path to the watermark image.
+- ``watermark_position``: the watermark position (scale or tile)
```

Comment:
From the code it seems that it is also possible to specify the position with a tuple which is given to http://pillow.readthedocs.org/reference/Image.html#PIL.Image.Image.paste
Could you also document this behavior ?